codespell_lib/data/dictionary.txt

```diff
@@ -32517,6 +32537,8 @@ incremet->increment
 incremetal->incremental
 incremeted->incremented
 incremeting->incrementing
+incremint->increment
+incremment->increment
```

Comment:
I would recommend adding all verbal forms, if you have the time and patience.

Reply:
Always short on time, but it's not too long.
Added a bonus for another similar case.